website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -10,7 +10,7 @@ description: Instructions for migrating a Go-based project built prior to `v1.0.
 
 The v1.0 release improves upon prior `operator-sdk` releases with a new project structure and CLI, each of which enhances project extensibility and customizability. These design changes are influenced by [`kubebuilder`](https://book.kubebuilder.io/).
 
-**Note:** It is recommended that you have your project upgraded to the latest SDK release version (0.19.x+) before following the steps in this guide to migrate to the new layout. However, the steps might work from previous versions as well. In this case, if you find an issue which is not covered here then check the previous [Migration Guides][migration-doc] which might help out.
+**Note:** It is recommended that you have your project upgraded to the latest SDK release version (0.19.x+) before following the steps in this guide to migrate to the new layout. However, the steps might work from previous versions as well. In this case, if you find an issue which is not covered here then check the previous [Migration Guides][migration-doc] which might help out. Also, to understand how to use the instructions from some links provided check the [FAQ: Can I use the Kubebuilder docs?][faq-can-I-use-kb-docs].
```

Comment:
This is the only one that actually makes sense and does not feel out of place.

Reply:
So, only in the migration guides or only in the golang migration guide? 
The ansible/helm migration also redirects the users to kb docs because of the multi-group scenario and metrics. 

So, I will keep for now only the faq + migration docs and I will wait for your feedback. 
Really thank you for your time.